docs/feature-flags/sdks/ios.md

```diff
@@ -49,9 +49,9 @@ Task {
 }
 ```
 
-It is reccomended to wrap initialization in a `Task` block in order to perform network request asynchronously.
```

Comment:
Typo fix. Should we be concerned that our iOS SDK doesn't do event logging?

Reply:
Like it doesn't log the assignment to the data warehouse? Yeah that sounds concerning 😛

Reply:
I don't see any logging code in the IOS SDK

Reply:
I'm going to leave event logging in iOS out of this doc update. I made a Linear ticket to get this implemented: https://linear.app/eppo/issue/FF-998/missing-logging-in-ios-sdk